tests/components/shopping_list/test_init.py

```diff
@@ -504,42 +506,50 @@ async def test_ws_reorder_items_failure(hass, hass_ws_client, sl_setup):
 
 async def test_add_item_service(hass, sl_setup):
     """Test adding shopping_list item service."""
-    await hass.services.async_call(
-        DOMAIN,
-        SERVICE_ADD_ITEM,
-        {ATTR_NAME: "beer"},
-        blocking=True,
-    )
-    await hass.async_block_till_done()
+    with patch("homeassistant.core.EventBus.async_fire") as mock_async_fire:
```

Comment:
There's a test helper `async_capture_events`, use that instead.

Reply:
Ah thanks!